recipes/apache-airflow-providers-alibaba/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "apache-airflow-providers-alibaba" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/apache-airflow-providers-alibaba-{{ version }}.tar.gz
+  sha256: 2407c1099ce49401084bf361068ebe7bf3ad9fde6e013f3e2a6fccb326c9d424
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - setuptools
+  run:
+    - python >=3.7
+    - apache-airflow >=2.1.0
+    - oss2 >=2.14.0
+
+test:
+  imports:
+    - airflow.providers.alibaba
+    - airflow.providers.alibaba.cloud
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://airflow.apache.org/
+  summary: Provider for Apache Airflow. Implements apache-airflow-providers-alibaba package
+  license: Apache-2.0
+  license_file: LICENSE
+  license_family: Apache
+  doc_url: https://airflow.apache.org/docs/{{ name }}/stable/index.html
```

Comment:
This is not a blocker but it's always good to hardcode these URLs as sometimes something trips up on this. Also applies to your other PRs but again, not a blocker.

Reply:
```suggestion
  doc_url: https://airflow.apache.org/docs/apache-airflow-providers-alibaba/stable/index.html
```